Dockerfile.apache

```diff
@@ -8,12 +8,20 @@
 # Contributors:
 #   Red Hat, Inc. - initial API and implementation
 
-FROM docker.io/httpd:2.4.43-alpine
+FROM docker.io/node:8.16.2-alpine as builder
```

Comment:
it would be nice to add in ci.yaml a build of --target=builder and publish image and pull that image
```
docker pull master-image || true
```


or use buildx to benefits from layers
so